pom.xml

```diff
@@ -55,6 +55,16 @@
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>aws-credentials</artifactId>
```

Comment:
Really do not want this dependency. If anything, `aws-credentials` could depend on the more generic `credentials-binding`. Anyway this is unrelated to the main thrust of the PR.